wiki/Tournaments/CMC/2020/en.md

```diff
@@ -0,0 +1,388 @@
+| Semifinals | 2020-11-30/2020-12-06 |
+| Finals week 1 | 2020-12-07/2020-12-13 |
+| Finals week 2 | 2020-12-14/2020-12-20 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 3 months of osu!supporter tag for each team member + 100% of the raised prize pool |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 2 months of osu!supporter tag for each team member |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter tag for each team member |
+| *4th place* | Steam Wallet voucher worth approx. IDR 50.000,- for each team member |
+
+## Organization
+
+Cemara Mania Cup \#1 (2020) was run by various Indonesian osu! community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ![][flag_ID] [-AMIYA-](https://osu.ppy.sh/users/7219949) |
+| Co-host | ![][flag_ID] [LLENN-](https://osu.ppy.sh/users/7428653) |
+| Mappool selector | ![][flag_ID] [Fyl](https://osu.ppy.sh/users/10069307), ![][flag_ID] [xChose](https://osu.ppy.sh/users/10213428), ![][flag_ID] [Kurami](https://osu.ppy.sh/users/10184990) |
+| Playtester | ![][flag_ID] [Fyl](https://osu.ppy.sh/users/10069307), ![][flag_ID] [xChose](https://osu.ppy.sh/users/10213428), ![][flag_ID] [Kurami](https://osu.ppy.sh/users/10184990) |
+| Streamer | ![][flag_ID] [-AMIYA-](https://osu.ppy.sh/users/7219949) |
+| Commentator | ![][flag_ID] [Fyl](https://osu.ppy.sh/users/10069307), ![][flag_ID] [Sies](https://osu.ppy.sh/users/6491991), ![][flag_ID] [\[ID\]Xenotia-](https://osu.ppy.sh/users/13865224), ![][flag_ID] [piesachan](https://osu.ppy.sh/users/3356184), ![][flag_ID] [Hwarry](https://osu.ppy.sh/users/2318309), ![][flag_ID] [-\[ ZeroTwo \]-](https://osu.ppy.sh/users/15052487), ![][flag_ID] [RizalRhmn](https://osu.ppy.sh/users/12532297) |
+| Referee | ![][flag_ID] [-AMIYA-](https://osu.ppy.sh/users/7219949), ![][flag_ID] [Fyl](https://osu.ppy.sh/users/10069307), ![][flag_ID] [xChose](https://osu.ppy.sh/users/8738085), ![][flag_ID] [Kurami](https://osu.ppy.sh/users/10184990), ![][flag_ID] [-Nana-](https://osu.ppy.sh/users/8738085), ![][flag_ID] [LyfLin](https://osu.ppy.sh/users/5198950) |
+| Graphic designer | ![][flag_ID] [-AMIYA-](https://osu.ppy.sh/users/7219949), ![][flag_ID] [-Nana-](https://osu.ppy.sh/users/8738085) |
+| Spreadsheet manager | ![][flag_ID] [-AMIYA-](https://osu.ppy.sh/users/7219949), ![][flag_ID] [Sies](https://osu.ppy.sh/users/6491991), ![][flag_ID] [Ririya-](https://osu.ppy.sh/users/15590443) |
+| Wiki editor | ![][flag_ID] [Niva](https://osu.ppy.sh/users/197805) |
+
+## Links
+
+- **[Main spreadsheet](https://docs.google.com/spreadsheets/d/1VttS6M8GdWINmMBuHws6AcND0yh3mNGzUiVQwleLP4Q/edit#gid=1467418977)**
+- [Forum thread](https://osu.ppy.sh/community/forums/topics/1157755)
+- [Discord server](https://discord.gg/5a2dy53)
+- [Livestream channel](https://www.twitch.tv/osuIndonesia)
+- [Challonge bracket](https://challonge.com/CMC_1)
```

Comment:
```suggestion
- **[Main spreadsheet](https://docs.google.com/spreadsheets/d/1VttS6M8GdWINmMBuHws6AcND0yh3mNGzUiVQwleLP4Q/edit#gid=1467418977 "Google Spreadsheet")**
- [Forum thread](https://osu.ppy.sh/community/forums/topics/1157755)
- [Discord server](https://discord.gg/5a2dy53 "Discord")
- [Livestream channel](https://www.twitch.tv/osuIndonesia "Twitch")
- [Challonge bracket](https://challonge.com/CMC_1 "Challonge")
```

Reply:
not sure if this is needed since i've never seen any other tournament article implementing this actually